### PR TITLE
Fixed the padding in portrait mode

### PR DESCRIPTION
**Pull request by mtropper**
#701 fixed the extra padding that was added in Portrait mode

### CODE REVIEW

File: src/screens/home/HomeScreen.tsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ export const HomeScreen = () => {
       <Box flex={1} alignItems="center" backgroundColor={strToBackgroundColor(backgroundColor)}>
         <Box
           flex={1}
-          padding="m"
+          paddingTop="m"
```
Comment (henrytao-me) [inline src/screens/home/HomeScreen.tsx:196]: It's better to add `paddingBottom="m"` too.

**Comment by mtropper** [comment]
added paddingBottom as well :)

**Comment by amazingphilippe** [comment]
I can also see that we have extra padding in OverlayView.tsx. 
line 153. we can just remove the two padding attributes!

Simulator Screen Shot - iPhone 11 - 2020-07-16 at 15 41 49

**Comment by mtropper** [comment]
@amazingphilippe the padding was added because in landscape part of the text gets covered. 
image

**Comment by mtropper** [comment]
@JuliannaR @amazingphilippe I modified it so the padding only appears in Landscape mode

**Pull request merged**